app/Resources/FOSUserBundle/serializer/Model.User.yml

```diff
@@ -0,0 +1,11 @@
+FOS\UserBundle\Model\User:
```

Comment:
@stloyd Yes, but if you look at all other directories in https://github.com/Sylius/Sylius/pull/2061/files#diff-0fdb2e4caadf1477ca2ccf6d90aa5fc4R118 they are in config.